### ISSUE TITLE
2 finger swipe up or down leads to blank pages

### ISSUE THREAD

**Issue by regrubtaf** [ddvk/remarkable-hacks#363]
Using the DDVK two finger gestures to switch documents sometimes opens the file up with blank pages.  So far, seems to only be a problem with large (100+ page pdfs and epubs).  If the file is opened up either through the file menu or the rM's native file preview system, the documents open up fine.  And once opened through a native method, the DDVK document switching methods work fine again for a while too.  Eventually, the problem reappears though.  Running. 2.10.1.332 and patch 26.2.02.  I can post a video of what I'm talking about if that would be helpful.

**Comment by martin-braun** [comment]
I can confirm this issue, it happens since 2.10.1.332, but it happens rarely. It also happens on notebooks with 30 pages or so. I installed it about a week ago and experienced this only 2 or 3 times since then, but in one instance it was so worse that I've chosen to restart my rM.

It got triggered by two finger swipe down to the last book. Right before I got this issue I also noticed that files open slower than normal. When I had the white page, I could swipe left and right again (going one page forward and back again) to fix this issue, but in one case it did not stop to happen so I had to reboot to get rid of it.
